app/components/container/new-edit/component.js

```diff
@@ -204,6 +205,16 @@ export default Component.extend(NewOrEdit, ChildHook, {
   validate() {
     let pr = get(this, 'primaryResource');
     let errors = pr.validationErrors() || [];
+    const lc = get(this, 'launchConfig');
+
+    const quotaErrors = lc.validateQuota() || [];
```

Comment:
validateQuota returns an empty array if `projectLimit` doesn't exist or an empty errors array. I think the extra defensive array here is unneeded.